modules/nextflow/src/main/groovy/nextflow/datasource/SraExplorer.groovy

```diff
@@ -292,6 +346,27 @@ class SraExplorer {
         xmlParser.parseText("<document>$xml</document>")
     }
 
+    protected parseCsv (String text) {
+        if (!text)
+            return null
+
+        def parsedCsv = [:]
+        def lines = text.trim().readLines()
+        def header = lines[0].tokenize('\t')
+        // def content =  lines[1].tokenize('\t')
+        String[] content = lines[1].split(/\t/)
+
+        header.eachWithIndex { key, index ->
+            def values = content[index].tokenize(';')
+            if ( key == 'fastq_ftp') {
+                values = values.each { str -> FileHelper.asPath("ftp://$str") }
```

Comment:
Values should be returned as strings (only the default implementation will continue to convert to Path).

Reply:
Multi-values attributes need to be tokenize to return an array with the corresponding values, that is why it was implemented this way. Otherwise, we can return the literal string with the comma. If we want to implement in this way, test should be modified.